src/middleware/handleJWTAndAddToContext.js

```diff
@@ -1,21 +1,34 @@
 const decode = require('jwt-decode');
+const { Forbidden } = require('@feathersjs/errors');
 
-const jwtHandler = (req, res, next) => {
+
+const jwtHandler = (feathers, authorization) => {
 	const regex = /Bearer (.+)/g;
-	const jwt = (regex.exec(req.headers.authorization) || [])[1]; // todo length >= 2 test.
+	const jwt = (regex.exec(authorization) || [])[1]; // todo length >= 2 test.
 
-	if (jwt) {
+	try {
```

Comment:
Catch at this position is danger, the middleware should only aggregate the data but not interpret it. Please revert it to if condition